packages/core/src/internal/decorators/event.ts

```diff
@@ -24,7 +24,7 @@ export class EventEmitter<T> {
 }
 
 // Legacy TS Decorator
-function legacyEvent(descriptor: PropertyDescriptor, protoOrDescriptor: {}, name: PropertyKey) {
+function legacyEvent(descriptor: PropertyDescriptor, protoOrDescriptor: Record<string, unknown>, name: PropertyKey) {
```

Comment:
There are a bunch of these. `Record<string, unknown>` is a stand in type for `{}`